app/models/appdata.rb

```diff
@@ -35,8 +35,13 @@ def self.add_appdata(data, xml)
 
   # Get the appdata xml for a distribution
   def self.get_distribution(dist = 'factory', flavour = 'oss')
-    appdata_url = if dist == "factory"
-                    "http://download.opensuse.org/tumbleweed/repo/#{flavour}/suse/setup/descr/appdata.xml.gz"
+    appdata_url = case dist
```

Comment:
why are you using a `case` here if there is only one option? :thinking: You can just use an `if-else` or do we expect to have more cases? :thinking:

Reply:
I don't know if there will be more cases, but the `case` read to me more explicit about "the url depends on the distro" than "do an exception" only for tumbleweed.

But I can change it, if it is generally considered bad style.

Reply:
Sorry, I looked at it once and again, and the case reads better :-)